Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -107,10 +107,8 @@ export class JavaFilesExtractor {
                 // 3 cleanup temp folder
                 console.log(taskLib.loc('RemoveTempDir', tempFolder));
                 taskLib.rmRF(tempFolder);
-            } else if ('.zip' === fileEnding) {
-                this.unzipExtract(file, this.destinationFolder);
             } else { // use sevenZip
-                this.sevenZipExtract(file, this.destinationFolder);
+                await this.sevenZipExtract(file, this.destinationFolder);
```

Comment:
sevenZipExtract() method seems to be calling execSync. Is there a reason for it to be async? Looks like we can remove the async and not need to await.

Reply:
If we want to unzip async, we should move to .exec() method on the tool and await on all the async methods. Looks like there is a mix and match which is very confusing.